test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Common/UserCommonOperation.cs

```diff
@@ -64,41 +68,53 @@ public static void UserSignOut(FirefoxPrivateVPNSession vpnClient)
             mainScreen.ClickSettingsButton();
             SettingScreen settingScreen = new SettingScreen(vpnClient.Session);
             Assert.AreEqual("Settings", settingScreen.GetTitle());
+            settingScreen.ScrollDown();
             settingScreen.ClickSignOutButton();
         }
 
         /// <summary>
         /// Send API request to check whether user is connected or not.
         /// </summary>
         /// <returns>The API response.</returns>
-        public static IRestResponse AmIMullvad()
+        /// <param name="expectedContent">Expected content returned from API.</param>
+        public static IRestResponse AmIMullvad(string expectedContent = null)
         {
             var client = new RestClient("https://am.i.mullvad.net/connected");
             var request = new RestRequest(Method.GET);
-            request.AddHeader("cache-control", "no-cache");
-            request.AddHeader("Connection", "keep-alive");
-            request.AddHeader("Accept-Encoding", "gzip, deflate");
-            request.AddHeader("Host", "am.i.mullvad.net");
-            request.AddHeader("Cache-Control", "no-cache");
-            request.AddHeader("Accept", "*/*");
-            return RetryExecute(client, request);
+            Func<IRestResponse, bool> condition = (res) =>
+            {
+                if (string.IsNullOrEmpty(expectedContent))
+                {
+                    return res.StatusCode != HttpStatusCode.OK;
+                }
+
+                return res.StatusCode != HttpStatusCode.OK || !res.Content.Contains(expectedContent);
+            };
+            IRestResponse response = RetryExecute(client, request, condition);
+            return response;
         }
 
         /// <summary>
         /// Call Mullvad API to get current city.
         /// </summary>
         /// <returns>The API response.</returns>
-        public static IRestResponse GetCityViaMullvad()
+        /// <param name="expectedCity">Expected city returned from API.</param>
+        public static IRestResponse GetCityViaMullvad(string expectedCity = null)
         {
             var client = new RestClient("https://am.i.mullvad.net/city");
```

Comment:
Should eventually offload these to a const block on the top of this file or in a separate file.